portfolio/src/main/webapp/comments.js

```diff
@@ -1,3 +1,5 @@
+const COMMENTSLIST = document.getElementById('comments-list');
```

Comment:
You should use COMMENTS_LIST since its two words. You could also call it COMMENTS_LIST_DOC_ELEMENT, to be more clear.